sdk/storage/azure-storage-blob-nio/src/test/java/com/azure/storage/blob/nio/APISpec.groovy

```diff
@@ -372,13 +372,17 @@ class APISpec extends Specification {
         }
     }
 
-    Map<String, Object> initializeConfigMap() {
+    Map<String, Object> initializeConfigMap(HttpPipelinePolicy... policies) {
         def config = [:]
         config[AzureFileSystem.AZURE_STORAGE_HTTP_CLIENT] = getHttpClient()
+        def policyList = []
```

Comment:
Should we initialize `policyList` using `def policyList = [] as HttpPipelinePolicy[]` instead of casting it later?